### ISSUE TITLE
Support more than one repo

### ISSUE THREAD

**Issue by streetturtle** [streetturtle/awesome-wm-widgets#143]
Support more than one repo

**Comment by Ryuno-Ki** [comment]
In what sense? Breaking this apart into smaller repo? Or have one widget being able to be instantiated multiple times?

The label says Bitbucket. So I guess it would be about a specific widget?

**Comment by streetturtle** [comment]
Yep, this issue is related to the bitbucket widget which is showing assigned pull requests. But only from one bitbucket repository.
